third_party/amd/include/TritonAMDGPUToLLVM/TargetUtils.h

```diff
@@ -19,6 +19,16 @@ enum class ISAFamily {
 // Deduces the corresponding ISA family for the given target gfx |arch|.
 ISAFamily deduceISAFamily(llvm::StringRef arch);
 
+// Here is a partial definition of DppCtrl enums. For the complete definition,
+// please check:
+// https://github.com/llvm/llvm-project/blob/llvmorg-19.1.3/llvm/lib/Target/AMDGPU/SIDefines.h
+enum DppCtrl : uint32_t {
```

Comment:
`enum class` so we make sure the fields are namespaced under it given this is in a header file.